frontend/src/app/feedback/page.tsx

```diff
@@ -46,7 +46,7 @@ export default function FeedbackPage() {
 
     try {
       const response = await fetch(
-        "http://localhost:5001/schemessg-v3-dev/asia-southeast1/feedback",
+        `${process.env.NEXT_PUBLIC_API_BASE_URL}/${process.env.NEXT_PUBLIC_API_PROJECT}/${process.env.NEXT_PUBLIC_API_REGION}/feedback`,
```

Comment:
An example staging endpoint from backend is 
https://asia-southeast1-schemessg-v3-dev.cloudfunctions.net/health
https://asia-southeast1-schemessg-v3-dev.cloudfunctions.net/schemes/1iOElk5UL9WkdLSbGVZO
https://asia-southeast1-schemessg-v3-dev.cloudfunctions.net/chat_message
https://asia-southeast1-schemessg-v3-dev.cloudfunctions.net/schemes_search

Your formatting here doesn't work for staging endpoints.

Reply:
perhaps keep things simple like
`.env.development`
NEXT_PUBLIC_API_BASE_URL=http://0.0.0.0:5001/schemessg-v3-dev/asia-southeast1

`.env.staging`
NEXT_PUBLIC_API_BASE_URL=https://asia-southeast1-schemessg-v3-dev.cloudfunctions.net

Reply:
okay can